### DISCUSSION TITLE
Windows

### DISCUSSION THREAD

**Discussion by leonidcan** [ericaltendorf/plotman#473] [General]
I would like to use  Windows  but when I try to configure it's says readline package not installed, it seems that this packages is for *nix/mac only? Can you please direct me in right direction? Thanks!

**Comment by altendky** [comment]
For now the suggestion is to use WSL.  Note https://github.com/ericaltendorf/plotman/issues/108#issuecomment-822151350.  We are working on pieces to move towards directly supporting Windows.

**Reply by leonidcan** [reply]
Does it mean I have to run chia in WSL? What will be faster windows native or in *nix container? I feel that native Windows Chia should be faster? Thank you!

**Reply by altendky** [reply]
I don't know the details.  I've heard that WSL1 is faster than native Windows for plotting and WSL2 is slower unless you do per the link I shared then...  I forget, WSL2 might have been faster than Windows native.  As a detail, I'm not sure if 'container' is a proper term for WSL or not.
